package.json

```diff
@@ -40,5 +51,42 @@
   },
   "dependencies": {
     "mobx": "^5.9.0"
+  },
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/cz-conventional-changelog"
+    }
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  },
+  "release": {
+    "branch": "chore/sl-83/prism-packaging-and-binary-deployment",
+    "verifyConditions": [
+      "@semantic-release/git",
+      "@semantic-release/github",
```

Comment:
Remove

Reply:
Do we need `@semantic-release/commit-analyzer`?

Reply:
I reckon we do for stuff like "BREAKING CHANGE" etc.

Reply:
Looks like might be already included actually? https://github.com/Updater/semantic-release-monorepo#release-config - search that section for "set of plugins it wraps".

Reply:
Let me check, I know recall that the `monorepo-semantic-release` re-wrapped plugins and maybe that's why.